shared/clientstats/scrapers_test.go

```diff
@@ -0,0 +1,183 @@
+package clientstats
+
+import (
+	"encoding/json"
+	"io"
+	"net/http"
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/prysmaticlabs/prysm/shared/testutil/assert"
+)
+
+type mockRT struct {
+	body       string
+	status     string
+	statusCode int
+}
+
+func (rt *mockRT) RoundTrip(req *http.Request) (*http.Response, error) {
+	return &http.Response{
+		Status:     http.StatusText(http.StatusOK),
+		StatusCode: http.StatusOK,
+		Body:       io.NopCloser(strings.NewReader(rt.body)),
+	}, nil
+}
+
+var _ http.RoundTripper = &mockRT{}
+
+func TestBeaconNodeScraper(t *testing.T) {
+	bnScraper := beaconNodeScraper{}
+	bnScraper.tripper = &mockRT{body: prometheusTestBody}
+	r, err := bnScraper.Scrape()
+	if err != nil {
```

Comment:
Can be more consistent with rest of the codebase by doing

`require.NoError(t, err, Unexpected error calling beaconNodeScraper.Scrape)`

Reply:
Updated, although I just realized I've been using `assert.NoError` in multiple places rather than `require.NoError`. I see the difference is that require uses `FatalF` and assert uses `Errorf`. I'll put more care into picking between assert/require going forward.